### PR TITLE
SOC-1303 Add i18n + apply styling fixes + add type

### PR DESCRIPTION
**Pull request by armonr**
/CC @garthwebb @kvas-damian @mklucsarits 

https://wikia-inc.atlassian.net/browse/SOC-1303

### CODE REVIEW

File: front/styles/main/module/_discussion.scss [hunk 1]
```diff
@@ -112,7 +112,12 @@
 		display: inline-block;
 		font-size: 10px;
 		font-weight: bold;
+	  	margin: 0;
 		width: 100%;
+
+	  	.icon {
+		  margin-right: 3px;
```
Comment (mklucsarits) [inline front/styles/main/module/_discussion.scss]: Make sure to use tabs for indentation. Otherwise this all LGTM.

**Comment by garthwebb** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

File: front/templates/main/discussion/post.hbs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	{{#if canShowMore}}
         <button {{action 'expand'}} class="see-more">
 			{{svg 'comments' viewBox='0 0 30 30' class='icon replies'}}
-            View Older Replies
+            {{i18n 'discussion.pagination-post-show-more'}}
```
Comment (rogatty) [inline front/templates/main/discussion/post.hbs]: Tabs here.

**Comment by armonr** [comment]
@garthwebb, @bekcunning : Added the i18n updates per [LOC-173](https://wikia-inc.atlassian.net/browse/LOC-173)

**Comment by hakubo** [comment]
lGTM

**Pull request merged**